### PR TITLE
[AI] Cleanup research

### PR DESCRIPTION
**Pull request by Cjkjvfnby**
A couple of small improvement, it's better to check code commit by commit. 

@geoffthemedio , could you add @Grummel7 to the project, so I could request a review from him?

### CODE REVIEW

File: default/python/AI/ResearchAI.py [hunk 1]
```diff
@@ -704,36 +702,28 @@ def exclude_tech(tech_name):
...
     """generate research orders"""
-    report_adjustments = False
     empire = fo.getEmpire()
     empire_id = empire.empireID
     aistate = get_aistate()
     enemies_sighted = aistate.misc.get("enemies_sighted", {})
     galaxy_is_sparse = ColonisationAI.galaxy_is_sparse()
-    debug("Research Queue Management:")
+
+    empire = fo.getEmpire()
```
Comment (Grummel7) [inline default/python/AI/ResearchAI.py]: That's a duplicate of line 705

File: default/python/AI/ResearchAI.py [hunk 2]
```diff
@@ -849,10 +839,7 @@ def generate_classic_research_orders():
                 ):
                     res = fo.issueEnqueueTechOrder(cc_tech, insert_idx)
                     msg = "Empire is very aggressive, so attempted to fast-track %s, got result %d" % (cc_tech, res)
-                    if report_adjustments:
-                        chat_human(msg)
-                    else:
-                        debug(msg)
+                    debug(msg)
```
Comment (Grummel7) [inline default/python/AI/ResearchAI.py]: You may as well remove msg and simply call:
`debug("Empire is very aggressive, so attempted to fast-track %s, got result %d" % (cc_tech, res))`

Same for all the other cases of course.

**Review by Grummel7 — CHANGES_REQUESTED** [review]
If I ever want to use chat_human like this, I'll put it into a function that that debug and if enabled also chat_human. I don't like if ... else blocks copied all over the place like this.

I think it definitely simplifies the code without removing anything useful. Just go all the way and remove the unneeded msg definitions as well.

File: default/python/AI/ResearchAI.py (hunk 2 above)
Comment (Cjkjvfnby) [reply default/python/AI/ResearchAI.py]: This operation is called inline. It's a part of the refactoring menu in the IDE.

File: default/python/AI/ResearchAI.py [hunk 3]
```diff
@@ -930,14 +908,13 @@ def generate_classic_research_orders():
...
                         res = fo.issueEnqueueTechOrder(supply_tech, num_techs_accelerated)
-                        msg = (
-                            "Empire has poor colony/outpost prospects, so attempted to fast-track %s, got result %d"
-                            % (supply_tech, res)
+                        debug(
+                            (
+                                "Empire has poor colony/outpost prospects, so attempted to fast-track %s, got result %d",
+                                supply_tech,
+                                res,
+                            )
```
Comment (Grummel7) [inline default/python/AI/ResearchAI.py:916]: I'm surprised that neither pycharm nor black handles that:
You could remove one pair of brackets here. Same in line 960 and 1145.

File: default/python/AI/ResearchAI.py [hunk 4]
```diff
@@ -930,14 +908,13 @@ def generate_classic_research_orders():
...
                         )
```
Comment (Grummel7) [inline default/python/AI/ResearchAI.py]: I'm surprised that neither pycharm nor black handles that:
You could remove one pair of brackets here. Same in line 960 and 1145.
(updated code reference)

**Review by Grummel7 — APPROVED** [review]

**Pull request merged**